var/spack/repos/builtin/packages/xpmem/package.py

```diff
@@ -39,6 +39,10 @@ class Xpmem(AutotoolsPackage):
     # Added RHEL 8.3 kernel support
     # Here 2.6.5-36 referes to 2.6.5 version and 36th commit id
     patch('xpmem_v2.6.5-36.patch', when="@2.6.5-36", level=1)
+    patch('https://github.com/hjelmn/xpmem/commit/cbd6e5bd3d2a1d3823c335ddcd3c57b94474f578.patch?full_index=1',
+          sha256='75299398b6c15546479bfbb8aa972431f58637fe2f0328196a26738bd7148140')
+    patch('https://github.com/hjelmn/xpmem/commit/7d346aaf1fdfc24d38cebb4ad107b7f5c43769e9.patch?full_index=1',
+          sha256='6be8c5f33d55c611924d8412253740f6f4b738e6d98e32981fa300d2ccbe99cc')
```

Comment:
Should these patches be applied for all versions of this package? Since they are commits, I'd assume that `when=@:2.6.5-36` would make sense (I assume the next version will include these commits).

Reply:
@scheibelp I am not sure if they are needed/will work with older versions, so I'd prefer to tag them with exact version. This package did not have a proper release for a long time (although the project looks active)...